stage_descriptions/base-03-wy1.md

```diff
@@ -1,9 +1,21 @@
-In this stage, you'll respond to multiple
-[PING](https://redis.io/commands/ping) commands sent by the same connection.
+In this stage, you'll respond to multiple [PING](https://redis.io/commands/ping) commands sent by the same connection.
+
+### Handling Multiple Commands
 
 A Redis server starts to listen for the next command as soon as it's done responding to the previous one. This allows
 Redis clients to send multiple commands using the same connection.
 
+For this stage, your server should run a loop that continuously reads commands and sends responses back through the same connection. Since we're only dealing with the `PING` command for now, your program can stay in a loop that reads input and responds with `+PONG\r\n`.
```

Comment:
```suggestion
For this stage, your server should run a loop that continuously reads commands and sends responses back through the same connection. Since we're only dealing with the `PING` command for now, your program can run a loop that reads input and responds with `+PONG\r\n`.
```

("stay in a loop" is a bit of weird usage)